app/views/home/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="greeting container flex">
```

Comment:
Please use `Proxima Nova Light` font for general text and `Proxima Nova Bold` for headings as defined in the design page in the whole app.